scripts/wrap_release.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python
+#
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS-IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Script that performs task to wrap up the release."""
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import getpass
+import os
+import re
+import sys
+
+import feconf
+
+from . import common
+
+_PARENT_DIR = os.path.abspath(os.path.join(os.getcwd(), os.pardir))
+_PY_GITHUB_PATH = os.path.join(_PARENT_DIR, 'oppia_tools', 'PyGithub-1.43.7')
+sys.path.insert(0, _PY_GITHUB_PATH)
+
+# pylint: disable=wrong-import-position
+import github # isort:skip
+# pylint: enable=wrong-import-position
+
+
+def remove_release_labels(repo):
+    """Removes PR: released labels from PRs.
+
+    Args:
+        repo: github.Repository.Repository. The PyGithub object for the repo.
+
+    Raises:
+        Exception: If any PR is not released.
+    """
+    current_release_label = repo.get_label('PR: for current release')
+    released_label = repo.get_label('PR: released')
+
+    current_release_prs = list(repo.get_issues(
+        state='all', labels=[current_release_label])) + list(
+            repo.get_issues(
+                state='open', labels=[released_label]))
+
+    if current_release_prs:
+        raise Exception(
+            'Following PRs are not released: %s.' % (
+                [pr.number for pr in current_release_prs]))
+
+    released_prs = repo.get_issues(state='closed', labels=[released_label])
+    for pr in released_prs:
+        pr.remove_from_labels('PR: released')
```

Comment:
Perhaps print to the terminal something like "Labels removed from PRs: #A, #B, ..." -- that way, if this is run in error, there is a (manual) undo mechanism.

Reply:
Done.